src/fitnesse/FitNesseExpediter.java

```diff
@@ -69,13 +69,9 @@ public void run() {
   }
 
   @Override
-  public void send(byte[] bytes) {
-    try {
-      output.write(bytes);
-      output.flush();
-    } catch (IOException e) {
-      LOG.log(Level.FINE, format("Could not send data for URL %s: %s (Stop button pressed?)", (request != null ? request.getResource() : ""), e.toString()));
```

Comment:
I see this message in my console quite often. I believe not only when I pressed stop, or navigated away from the page, but I have not been able to determine when it happens. 
Stopping a test when this occurs seems like a 'risky' change...